manifest.json

```
@@ -207,7 +207,7 @@
     },
     {
       "description": "Adds Treesitter syntax highlighting support",
-      "version": "0.1",
+      "version": "0.2",
       "remote": "https://github.com/TorchedSammy/Evergreen.lxl:c4058bfdedd27aee3b23319905b5df7b4ca52ef8",
```

Also the commit hash needs to be updated to that of the latest plugin version:

```suggestion
"remote": "https://github.com/TorchedSammy/Evergreen.lxl:e5e6ade897a6316fbe6fc610520d195e27737912",
```